content/app-dev/secure/_index.en.md

```diff
@@ -2,7 +2,7 @@
 title: "Securing a wasmcloud App"
 date: 2018-12-29T11:02:05+06:00
 weight: 7
-draft: false
+draft: true

```

Comment:
If you leave `draft` on here, I know a bunch of hugo templates won't actually render the output. Does ours render drafts?

Reply:
It does for me, which is why I changed it to see if it wouldn't render, but I'll revert this just in case the live site will hide it.